clients/unshaded/src/main/java/tachyon/client/ClientOptions.java

```diff
@@ -35,8 +34,8 @@
     private TachyonStorageType mTachyonStorageType;
     /** How this operation should interact with the under storage */
     private UnderStorageType mUnderStorageType;
-    /** Worker location to execute this operation, if not possible, the operation will fail */
-    private NetAddress mLocation;
+    /** Worker location to write data, if not possible, the operation will fail */
```

Comment:
In the current implementation, this doc is correct. Do you mind making it also work for reading data? Though we can do that in a separate PR. (Opened Ticket 1041) It would be modifying `TachyonBlockStore#getInStream` to also take location as an argument and doing the right thing there.